Lombiq.HelpfulLibraries.AspNetCore/Extensions/ConfigurationSectionExtensions.cs

```diff
@@ -18,4 +18,21 @@ public static IConfigurationSection AddValueIfKeyNotExists(
         configurationSection[key] ??= value;
         return configurationSection;
     }
+
+    /// <summary>
+    /// Creates a new instance of <typeparamref name="T"/> and binds it the provided configuration.
+    /// </summary>
+    /// <param name="configuration">The base configuration.</param>
+    /// <param name="sectionKey">
+    /// If <see langword="null"/> or empty, then <paramref name="configuration"/> is used, otherwise the section with
+    /// this key inside it.
+    /// </param>
+    public static T BindNew<T>(this IConfiguration configuration, string sectionKey = null)
```

Comment:
There is already something like this and `OrchardCore.Environment.Shell.Configuration.ShellConfigurationExtensions` in `Lombiq.HelpfulLibraries.OrchardCore.Mvc.ServiceCollectionExtensions`, please do something with this duplicate.